lib/report-portal-client.js

```diff
@@ -39,6 +41,8 @@ class RPClient {
             baseURL: this.baseURL,
             headers: this.headers,
         });
+        this.analytics = new Analytics();
+        this.analytics.setPersistentParams(params.token);
```

Comment:
Are you sure you want to share user authorization token with third party service?
maybe hash it before by md5\sha-256  at least?

Reply:
This field is superfluous in general. I removed it.

Reply:
👍 
Don't forget remove it from method call :)

Reply:
My omission, thank you :)